src/views/directory/directory.module.css

```
@@ -0,0 +1,306 @@
+  }
+
+  .flashListing {
+    margin-top: 10px;
+  }
+
+  .actionsCell {
+    white-space: nowrap;
+  }
+}
+
+:global(body.yotsuba) .rowOdd td {
```

Stylelint (17.11.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)